src/electionguard_cli/readme.md

```diff
@@ -0,0 +1,16 @@
+# Setup
```

Comment:
This should actual be in docs folder.

They are deployed to here:
https://pypi.org/project/electionguard
https://microsoft.github.io/electionguard-python/
https://electionguard-python.readthedocs.io/en/latest/

Reply:
Deleted the readme, as discussed